helm/korifi/values.schema.json

```diff
@@ -365,6 +365,10 @@
           "description": "The time that the kpack Builder will be waited for if not in ready state, berfore the build workload fails. See [`time.ParseDuration`](https://pkg.go.dev/time#ParseDuration) for details on the format, an additional `d` suffix for days is supported.",
           "type": "string"
         },
+        "clusterStackID": {
+          "description": "The ID of the `ClusterStack`. Used when `kpack-image-builder` is blank.",
```

Comment:
@kieron-dev I made `clusterStackID` configurable, but was a bit confused by the `"Used when `kpack-image-builder` is blank."` message we have in the description for the other properties. What does that mean?

Reply:
We discussed this on Slack, this should actually say `clusterBuilderName` instead.